src/renderer/GlobalLoader.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import {useSelector} from "react-redux";
+import {isApplicationLoading, isLoadingValidator} from "./ducks/settings/selectors";
+import {Loading} from "./components/Loading/Loading";
+
+export const GlobalLoader: React.FC = () => {
+    const isInitializing = useSelector(isApplicationLoading);
+    const isLoading = useSelector(isLoadingValidator);
+
+    return <Loading visible={isInitializing || isLoading} title='Loading...' />;
```

Comment:
do we wanna add support for canceling loader and actions? How would you implement it?

Reply:
i dont think we want to add option to cancel loading app on initial loading or storing validator in db